src/main/kotlin/no/nav/syfo/infrastructure/cronjob/PublishMQCronjob.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.infrastructure.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.VedtakService
+import org.slf4j.LoggerFactory
+
+class PublishMQCronjob(
+    private val vedtakService: VedtakService,
+) : Cronjob {
+    override val initialDelayMinutes: Long = 2
+    override val intervalDelayMinutes: Long = 1
+
+    override suspend fun run() {
+        val (success, failed) = vedtakService.sendVedtakToInfotrygd().partition { it.isSuccess }
+        failed.forEach {
+            log.error("Exception caught while publishing vurdering", it.exceptionOrNull())
+        }
+        if (failed.size + success.size > 0) {
```

Comment:
Fin løsning! Dette burde vi kanskje ha på alle cronjobbene våre? 🤔

Reply:
Enig. Kunne kanskje refaktorert ut til noe felles i `Cronjob` for dette senere :+1: